site/docs/how-to-release.md

```diff
@@ -204,7 +204,7 @@ The release tarball, signature, and checksums are here:
 * https://dist.apache.org/repos/dist/dev/iceberg/apache-iceberg-<VERSION>-rc<NUM>/
 
 You can find the KEYS file here:
-* https://dist.apache.org/repos/dist/dev/iceberg/KEYS
+* https://dist.apache.org/repos/dist/release/iceberg/KEYS
```

Comment:
I thought we wanted to use Downloads here (I assume this is actually fixed int the script)